crates/database/src/ethflow_orders.rs

```diff
@@ -0,0 +1,83 @@
+use crate::{OrderUid, PgTransaction};
+use sqlx::PgConnection;
+
+#[derive(Clone, Debug, Default, sqlx::FromRow, Eq, PartialEq)]
+pub struct EthOrderPlacement {
+    pub uid: OrderUid,
+    pub valid_to: i64,
+}
+
+pub async fn append(
+    ex: &mut PgTransaction<'_>,
+    events: &[EthOrderPlacement],
+) -> Result<(), sqlx::Error> {
+    for event in events {
+        insert_ethflow_order(ex, event).await?;
+    }
```

Comment:
Here it could be nice as well to have a function which is able to insert multiple orders at once.